operator/provider_aws_flags.go

```diff
@@ -29,6 +29,9 @@ func init() {
 	flags.Bool(operatorOption.AWSReleaseExcessIPs, false, "Enable releasing excess free IP addresses from AWS ENI.")
 	option.BindEnv(operatorOption.AWSReleaseExcessIPs)
 
+	flags.Int(operatorOption.ExcessIPReleaseDelay, 180, "Number of seconds operator would wait before it releases an IP previously marked as excess. Defaults to 180")
+	option.BindEnv(operatorOption.ExcessIPReleaseDelay)
+
```

Comment:
I think the `flags` package will actually already show the default value in the text string, so no need to add it yourself.

Reply:
Removed the default part.